modules/core/src/main/java/org/apache/ignite/mxbean/IgniteMXBean.java

```diff
@@ -515,4 +516,32 @@ void runIoTest(
     @MXBeanParametersNames("registry")
     @MXBeanParametersDescriptions("Metrics registry.")
     public void resetMetrics(String registry);
+
+    /**
+     * Checks cluster state.
+     *
+     * @return String representation of current cluster state.
+     * See {@link ClusterState}.
+     */
+    @MXBeanDescription("Checks cluster state.")
+    public String clusterState();
+
+    /**
+     * Changes current cluster state.
+     *
+     * @param state String representation of new cluster state.
+     * See {@link ClusterState}
+     */
+    @MXBeanDescription("Changes current cluster state.")
+    @MXBeanParametersNames("state")
+    @MXBeanParametersDescriptions("New cluster state.")
+    public void clusterState(String state);
+
+    /**
+     * Gets last cluster state change operation.
+     *
+     * @return Unix time of last cluster state change operation.
```

Comment:
what is : unix time ? does we have windows and apple time too ?

Reply:
It's a common phrase  https://en.wikipedia.org/wiki/Unix_time